selenium-system-test/src/test/scala/htsstride/pages/eligibility/SCRCustomerEligiblePage.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package htsstride.pages.eligibility
+
+import htsstride.browser.Browser
+import htsstride.pages.{ApplicationCancelledPage, CreateAccountPage, Page}
+import htsstride.utils.{Configuration, CustomerDetails}
+import org.openqa.selenium.WebDriver
+
+object SCRCustomerEligiblePage extends Page {
```

Comment:
this page and `ErrorSCREligiblePage` are actually the same page - one is just the page with errors on it. You could just stick the `findErrorMessageList` method in this object - that's the only different between the two objects.